resources/views/form.blade.php

```diff
@@ -72,6 +72,22 @@ class="form-control @error('command') is-invalid @enderror"
         <code>{{ trans('schedule::schedule.messages.attention-type-function') }}</code>
     </div>
 
+    <div v-if="form.command == 'custom'" class="form-group">
```

Comment:
I believe that the options in the command field lacked the option `custom`.
Or did I misunderstand how to select a custom field?